cornflow-dags/DAG/update_all_schemas.py

```diff
@@ -34,30 +33,67 @@ def get_new_apps() -> List[ApplicationCore]:
     return [app_class() for app_class in new_apps]
 
 
+def _load_ignore_patterns():
+    """
+    Load ignore patterns from .dagignore if it exists, else use defaults. Always include default patterns.
+    """
+    default_patterns = [
+        "scripts",
+        "documentation",
+        "tests",
+        "activate_dags",
+        ".",
+        "__"
+    ]
+
+    _dir = os.path.dirname(__file__)
+    dagignore_path = os.path.join(_dir, ".dagignore")
+
+    ignore_patterns = set(default_patterns)
+
+    if os.path.exists(dagignore_path):
+        try:
+            with open(dagignore_path, 'r') as f:
+                for line in f:
+                    line = line.strip()
+                    if line and not line.startswith('#'):
+                        ignore_patterns.add(line)
+            print(f"Loaded ignore patterns from {dagignore_path}")
+        except Exception as e:
```

Comment:
I guess the error here should be something like `FileNotFoundError`, no? I would prefer if the only expcetion that we capture is that one.